tests/test_bedtech.py

```
@@ -297,17 +297,17 @@ async def test_has_pillow_support(
 
         assert coordinator.controller.has_pillow_support is True
 
-    async def test_supports_stop_all_is_true(
+    async def test_supports_stop_all_is_false(
         self,
         hass: HomeAssistant,
         mock_bedtech_config_entry,
         mock_coordinator_connected,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)